app/views/shared/_header.html.erb

```diff
@@ -22,7 +22,7 @@
 </nav>
 <div class="brand">
   <h1 class="py-2 px-4 my-0 text-white">Accessioning</h1>
-  <%= image_tag("bookheader.png", width: "100%", height: "75") %>
+  <%= image_tag("bookheader.png", width: "100%", height: "75", alt: "") %>
```

Comment:
Blank?

Reply:
Oh, now I see the reason in the ticket. Can you add a comment so that a future dev doesn't think it is cruft?